Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml

```diff
@@ -32,10 +32,9 @@
                     If you know the person who registered the organisation, they can invite you to join. Or you can request to join below.
                 </p>
 
-                <govuk-button>
+                <a class="govuk-button" href="/registration/@Model.Identifier/join-organisation">
```

Comment:
If we're no longer posting a form with a button here, then the form on line 22 should go as well

Reply:
Will remove